src/styles/foundation/_spacing.scss

```diff
@@ -1,3 +1,4 @@
+// stylelint-disable-next-line scss/partial-no-import
```

Comment:
Why do you need these?

Reply:
Stylelint doesn't like it if you're in a partial file (i.e one starting with an _) and you then import other files.

I assume because of sass's clunky import system being fine with you importing the same file multiple times they want you to put all your imports in a single top-level file rather than depending on stuff on a per-file basis to make it clear you're not repeating yourself. If you put imports within imports it won't be clear if you accidentally import the same file twice.

We're choosing to say we know better and that in this constraint case we're fine as only this spacing file will import the spacing tokens (and the color file will import the color tokens etc).